src/Libraries/Microsoft.Extensions.Http.Resilience/Microsoft.Extensions.Http.Resilience.csproj

```diff
@@ -20,7 +19,7 @@
 
   <PropertyGroup>
     <Stage>normal</Stage>
-    <MinCodeCoverage>98</MinCodeCoverage>
+    <MinCodeCoverage>97</MinCodeCoverage>
```

Comment:
Should be at 100, but code coverage is reporting lower number :/

Reply:
It's been reported as
![image](https://github.com/dotnet/extensions/assets/4403806/92b4f0d3-45a6-43ed-9b0b-1036136c5550)

Because of this:
![image](https://github.com/dotnet/extensions/assets/4403806/16005908-89e1-4bb1-af07-7673734e845b)

Reply:
...though the validation script reports it as 97.38%...
![image](https://github.com/dotnet/extensions/assets/4403806/9b93a15e-667b-401e-a2a0-9bd4202a9cd3)